vr-je-palette-picker/palette-picker-front-end
# Frontend setup

## What does this PR do?
This PR installs dependencies for SCSS and testing, alongside creating a components directory that contains all components and their necessary styling and testing files.
### Where should the reviewer start?
Start in App.js to see initial class component structure, review packages installed in package.json file, finish up review in Components directory to see all files added.
#### How should this be manually tested?
Pull down, run `npm start` and see that Palette Picker is running in your local environment.
#### Any background context you want to provide?
This contains the simple file structure, components are being divvied up amongst project collaborators for the next MR.
#### What are the relevant tickets?
#2 - Not yet complete
#### Screenshots (if appropriate)
![White boarding session from 2019-12-07](https://user-images.githubusercontent.com/31895658/70381575-d5c3cb00-1909-11ea-9b51-989586c1d5a6.JPG)
#### Additional notes
Thank you for your hard work today!
